### PR TITLE
npm install

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!tastemetitus

File: scripts/install [hunk 1]
```diff
@@ -126,9 +126,7 @@ ynh_script_progression --message="Installing The Lounge..." --weight=10
 
 pushd $final_path
  	ynh_use_nodejs
- 	NODE_ENV=production ynh_exec_warn_less yarn install 
- 	NODE_ENV=production yarn add webpack webpack-cli --dev
- 	NODE_ENV=production ynh_exec_warn_less yarn build
+ 	$ynh_exec_as $app env $ynh_node_load_PATH ynh_npm install --global --unsafe-perm thelounge
```
Comment (tituspijean) [inline scripts/install]:
```suggestion
 	ynh_exec_as $app env $ynh_node_load_PATH ynh_npm install --global --unsafe-perm thelounge
```

😇

**Comment by alexAubin** [comment]
!testme

**Comment by tituspijean** [comment]
!gogogadgetotitus

File: conf/systemd.service [hunk 2]
```diff
@@ -7,10 +7,10 @@ Type=simple
 User=__APP__
 Group=__APP__
 WorkingDirectory=__FINALPATH__/
-Environment="PATH=__ENV_PATH__"
+Environment="__YNH_NODE_LOAD_PATH__"
 Environment="THELOUNGE_HOME=/home/yunohost.app/__APP__/"
 Environment="NODE_ENV=production"
-ExecStart=/usr/bin/yarn start
+ExecStart==thelounge start
```
Comment (tituspijean) [inline conf/systemd.service]:
```suggestion
ExecStart=thelounge start
```

**Pull request closed without merging**